packages/core/src/components/ColorPalette/ColorDisplay.tsx

```diff
@@ -99,6 +146,10 @@ function VariantRow({
       />
     </Box>
   )
+  function handleCopyColor(colorHex: string) {
+    navigator.clipboard.writeText(colorHex)
+    setHasCopiedHexCode(true)
```

Comment:
In the spirit of "colocation", let's just put this inline instead of separating it out to a function. It's only used in one place, so making its own function seems overkill.